ownCloud/Settings/DisplaySettingsSection.swift

```diff
@@ -24,12 +24,18 @@ class DisplaySettingsSection: SettingsSection {
 	override init(userDefaults: UserDefaults) {
 		super.init(userDefaults: userDefaults)
 
-		self.headerTitle = "Display settings".localized
+		self.headerTitle = "Advanced settings".localized
 
 		self.add(row: StaticTableViewRow(switchWithAction: { (row, _) in
 			if let newShowHiddenFiles = row.value as? Bool {
 				DisplaySettings.shared.showHiddenFiles = newShowHiddenFiles
 			}
 		}, title: "Show hidden files and folders".localized, value: DisplaySettings.shared.showHiddenFiles, identifier: "show-hidden-files-switch"))
+
+		self.add(row: StaticTableViewRow(switchWithAction: { (row, _) in
+			if let disableDragging = row.value as? Bool {
+				DisplaySettings.shared.dragFiles = disableDragging
+			}
+		}, title: "Drag files enabled".localized, value: DisplaySettings.shared.dragFiles, identifier: "drag-files-switch"))
```

Comment:
I think `Drag files enabled` could be confusing, as this feature is not limited to dragging files. I'd suggest to use `Drag & drop support` instead.

Reply:
This feature is limited to drag files. It is possible to drop files from outside, e.g. a third part app. The problem was, that users do not want to begin dragging files inside the ownCloud app.

Reply:
You're right. You can also drag folders within the app, though, which is not covered by the label for that option. Best solution probably is [what Michael suggested](https://github.com/owncloud/ios-app/pull/643#pullrequestreview-374150561): invert the meaning (`Enable` -> `Prevent` … so it's off by default, not on) and label it `Prevent dragging of files and folders`.

Reply:
@felix-schwarz  @michaelstingl thank you for your feedback. The new commit contains the changes!

Reply:
@felix-schwarz I will change the naming of the setting value, because it needs a new name after the changed wording.

Reply:
@felix-schwarz you can give it a new try. Finished from my side.